### PR TITLE
[Feature-2030][web] Add dashboard to display the platform overview

### LINKED ISSUE #2030: [Feature][web] Add dashboard to display the platform overview
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

_No response_

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by aiwenmo**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by aiwenmo** [comment]
a4c5b9b35fd5c3c6e7126b29bfa39b3
This is a static page, and I am implementing a data interface.

**Comment by aiwenmo** [comment]
I want to merge the code first to avoid unnecessary conflicts. @zhu-mingye

File: dinky-web/src/pages/Home/DevOverView/ResourceView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    <RcResizeObserver
+      key="resize-observer"
+      onResize={(offset) => {
+        setResponsive(offset.width < 296);
+      }}
+    >
+      <StatisticCard.Group direction={responsive ? 'column' : 'row'}>
+        <StatisticCard
+          statistic={{
+            title: 'Flink 集群实例',
```
Comment (Zzm0809) [inline dinky-web/src/pages/Home/DevOverView/ResourceView/index.tsx:44]: should be use I18N

File: dinky-web/src/pages/Home/DevOverView/ResourceView/index.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                style={imgStyle}
+                src="https://gw.alipayobjects.com/mdn/rms_7bc6d8/afts/img/A*dr_0RKvVzVwAAAAAAAAAAABkARQnAQ"
+                alt="icon"
+              />
+            ),
+          }}
+        />
+        <StatisticCard
+          statistic={{
+            title: '集群配置',
```
Comment (Zzm0809) [inline dinky-web/src/pages/Home/DevOverView/ResourceView/index.tsx:57]: should be use I18N

**Comment by Zzm0809** [comment]
> I want to merge the code first to avoid unnecessary conflicts. @zhu-mingye

first, the Chinese need change to i18n

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**